### ISSUE TITLE
Can you disable 3D models?

### ISSUE THREAD

**Issue by KuroCaos** [McJty/TheOneProbe#333]
I have a small problem with the 3D designs that appear in the MOD, I have other mods that add large mobs designs and when they appear in the mod they appear really large, making it look ugly and occupy part of the screen.

Is it possible to deactivate the 3D models that appear in the MOD? I already looked in the config and I didn't find something referring, I would like to ignore it by the type of name it bears

**Comment by Xhale1** [comment]
Yes please! If there could be a way to disable models for entities and blocks all together that would be great. It's just totally unnecessary to re-display what you are literally already looking at (plus the issues with large entities).

**Comment by P3rf3ctXZer0** [comment]
3d render breaking mods and making unnecessary work - please allow devs to shut it off.

**Comment by McJty** [comment]
I'll check it out

**Comment by McJty** [comment]
Actually this is already supported. Check out: theoneprobe-client.toml
There is this config:

#This is a list of entities that will not be rendered by TOP. This option also works if it is set client-side alone
renderBlacklist = []

You can put any mod you don't want to render in that list. You can also use * to disable them all like this:

renderBlacklist = ["*"]

**Comment by P3rf3ctXZer0** [comment]
This not fixed 
Crash Report https://mclo.gs/bmirMkC

@McJty 

Following your tutorial I crashed.
Config file
https://gist.github.com/P3rf3ctXZer0/27f63ffeeea290d02df5d0fa4c8c19df

**Comment by McJty** [comment]
Ah the wildcard is only supported for the blacklistEntities config. Not for renderBlacklist. You can still do it however but not with the wildcard. I'll see if I can add support for that but I no longer work on 1.16.5 so in 1.16.5 you will have to do it one by one
